NitroxServer/GameLogic/Players/PlayerData.cs

```diff
@@ -21,25 +21,31 @@ public class PlayerData
             {
                 lock (playersByPlayerName)
                 {
-                    return new Dictionary<string, PersistedPlayerData>(playersByPlayerName);
+                    serializablePlayersByPlayerName = new Dictionary<string, PersistedPlayerData>(playersByPlayerName);
+                    return serializablePlayersByPlayerName;
                 }
             }
             set { playersByPlayerName = value; }
         }
 
+        private Dictionary<string, PersistedPlayerData> serializablePlayersByPlayerName = new Dictionary<string, PersistedPlayerData>();
+
         [ProtoMember(2)]
         public Dictionary<NitroxId, EquippedItemData> SerializableModules
         {
             get
             {
                 lock (ModulesItemsById)
                 {
-                    return new Dictionary<NitroxId, EquippedItemData>(ModulesItemsById);
+                    serializableModules = new Dictionary<NitroxId, EquippedItemData>(ModulesItemsById);
+                    return serializableModules;
                 }
             }
             set { ModulesItemsById = value; }
         }
 
+        Dictionary<NitroxId, EquippedItemData> serializableModules = new Dicgionary<NitroxId, EquippedItemData>();
```

Comment:
There is a small spelling mistake, it should be "Dictionary"

Reply:
oof what i get for typin it on phone